scalyr_agent/platform_tests/alpine/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2-alpine
```

Comment:
Can you create a `README.md` file that describes the purpose of this directory (does platform level tests using docker to verify .. blah blah).